libs/ui/src/lib/image-upload/image-upload.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import {FullImageFragment} from '@wepublish/website/api'
+import {useRef} from 'react'
+import {MdDelete, MdEdit} from 'react-icons/md'
+import {IconButton} from '../icon-button/icon-button'
+
+export type ImageUploadProps = {
+  image?: FullImageFragment | null
+  onUpload: (image: any) => void
+}
+
+const Avatar = styled.img`
+  width: 150px;
+  height: 150px;
+  border-radius: 150px;
+`
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`
+
+const ActionButton = styled(IconButton)`
+  margin-left: 10px;
```

Comment:
Couldn't we use a grid on the parent? Else the sibling/child has to know about the layout.
Also please use the theme.

Note that this is not overrideable now in the website builder, as the IconButton is hardcoded and not taken from the builder (UI can not use the builder due to circular dependencies).

Would this maybe make sense to be in /user/website?